src/styles/fonts/fira-sans/fira.css

```diff
@@ -0,0 +1,221 @@
+       url('woff2/FiraSans-Bold.woff2') format('woff2'),
+       url('woff/FiraSans-Bold.woff') format('woff'),
+       url('ttf/FiraSans-Bold.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face{
+  font-family: 'Fira Sans';
+  src: url('eot/FiraSans-BoldItalic.eot');
+  src: url('eot/FiraSans-BoldItalic.eot') format('embedded-opentype'),
+       url('woff2/FiraSans-BoldItalic.woff2') format('woff2'),
+       url('woff/FiraSans-BoldItalic.woff') format('woff'),
+       url('ttf/FiraSans-BoldItalic.ttf') format('truetype');
+  font-weight: 700;
+  font-style: italic;
+}
+
+@font-face{
+  font-family: 'Fira Sans';
+  src: url('eot/FiraSans-ExtraBold.eot');
+  src: url('eot/FiraSans-ExtraBold.eot') format('embedded-opentype'),
+       url('woff2/FiraSans-ExtraBold.woff2') format('woff2'),
+       url('woff/FiraSans-ExtraBold.woff') format('woff'),
+       url('ttf/FiraSans-ExtraBold.ttf') format('truetype');
+  font-weight: 800;
+  font-style: normal;
+}
+
+@font-face{
+  font-family: 'Fira Sans';
+  src: url('eot/FiraSans-ExtraBoldItalic.eot');
+  src: url('eot/FiraSans-ExtraBoldItalic.eot') format('embedded-opentype'),
+       url('woff2/FiraSans-ExtraBoldItalic.woff2') format('woff2'),
+       url('woff/FiraSans-ExtraBoldItalic.woff') format('woff'),
+       url('ttf/FiraSans-ExtraBoldItalic.ttf') format('truetype');
+  font-weight: 800;
+  font-style: italic;
+}
+
+@font-face{
+  font-family: 'Fira Sans';
+  src: url('eot/FiraSans-Heavy.eot');
+  src: url('eot/FiraSans-Heavy.eot') format('embedded-opentype'),
+       url('woff2/FiraSans-Heavy.woff2') format('woff2'),
+       url('woff/FiraSans-Heavy.woff') format('woff'),
+       url('ttf/FiraSans-Heavy.ttf') format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+@font-face{
+  font-family: 'Fira Sans';
+  src: url('eot/FiraSans-HeavyItalic.eot');
+  src: url('eot/FiraSans-HeavyItalic.eot') format('embedded-opentype'),
+       url('woff2/FiraSans-HeavyItalic.woff2') format('woff2'),
+       url('woff/FiraSans-HeavyItalic.woff') format('woff'),
+       url('ttf/FiraSans-HeavyItalic.ttf') format('truetype');
+  font-weight: 900;
+  font-style: italic;
+}
+
+
+@font-face{
+  font-family: 'Fira Mono';
+  src: url('eot/FiraMono-Regular.eot');
+  src: url('eot/FiraMono-Regular.eot') format('embedded-opentype'),
+       url('woff2/FiraMono-Regular.woff2') format('woff2'),
+       url('woff/FiraMono-Regular.woff') format('woff'),
+       url('ttf/FiraMono-Regular.ttf') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face{
+  font-family: 'Fira Mono';
+  src: url('eot/FiraMono-Bold.eot');
+  src: url('eot/FiraMono-Bold.eot') format('embedded-opentype'),
+       url('woff2/FiraMono-Bold.woff2') format('woff2'),
+       url('woff/FiraMono-Bold.woff') format('woff'),
+       url('ttf/FiraMono-Bold.ttf') format('truetype');
+  font-weight: 600;
+  font-style: normal;
```

Comment:
Should these be loaded using [next/font/local](https://nextjs.org/docs/basic-features/font-optimization) most likely?

Having the full fonts locally is fine and all, but most of these are currently probably unused... Also I don't think we need to support anything other than `woff2`, as it's pretty [universally supported](https://caniuse.com/woff2), and recommended in the next.js docs?

Reply:
I remember I just imported the whole thing so I could quickly integrate it. I'm not sure what work I could do at the moment to integrate it and don't really have time time. We could just integrate it like this for now, or someone else could fix it.